infrastructure/code/issue_enhancer.go

```diff
@@ -233,6 +233,17 @@ func (b *IssueEnhancer) autofixFunc(ctx context.Context, issue snyk.Issue,
 	return editFn
 }
 
+func ToEncodedNormalizedPath(rootPath string, filePath string) (string, error) {
+	relativePath, err := ToRelativeUnixPath(rootPath, filePath)
+	if err != nil {
+		// couldn't make it relative, so it's already relative
+		relativePath = filePath
+	}
+
+	encodedRelativePath := EncodePath(relativePath)
+	return encodedRelativePath, nil
+}
+
```

Comment:
Maybe we could add some parameterized tests for this addition? Are these valid scenarios to test?
- `C:\\Users\\User\\project\\file.txt`
- `project\\file.txt`